src/inmanta/protocol/methods_v2.py

```diff
@@ -298,6 +307,8 @@ def environment_create_token(tid: uuid.UUID, client_types: List[str], idempotent
 )
 def environment_settings_list(tid: uuid.UUID) -> model.EnvironmentSettingsReponse:
     """
+    :param tid: The id of the environment for which the list of settings is being requested.
```

Comment:
Does this render properly in the API documentation if we put the parameters first and only the the description?

Reply:
I didn't check but I don't like it, it is not consistent with the rest. I will change it